components/Starknet/modules/architecture_and_concepts/pages/Smart_Contracts/builtins.adoc

```diff
@@ -0,0 +1,152 @@
+= Cairo Builtins
+
+== Introduction
```

Comment:
I don't think you actually need headings. This section is so small, and you can just use a table title instead of a heading for the table.